src/main/kotlin/com/velocitypowered/api/kt/util/Duration.kt

```diff
@@ -0,0 +1,13 @@
+package com.velocitypowered.api.kt.util
+
+import java.time.Duration
+import kotlin.time.ExperimentalTime
+import kotlin.time.Duration as KDuration
+
+fun Duration.ofTicks(ticks: Long): Duration =
```

Comment:
This isn't going to work, the `Duration` refers to a duration instance, not a companion object

Reply:
Likewise, the concept of ticks in Velocity doesn't make a lot of sense.